src/QY2Graph.cc

```diff
@@ -442,6 +439,51 @@ QY2Graph::renderGraph(graph_t* graph)
 }
 
 
+QString
+QY2Graph::unescape(const std::string& s) const
+{
+    std::string r;
+
+    bool backslashed = false;
+
+    for (const char c : s)
+    {
+	if (!backslashed)
+	{
+	    switch (c)
+	    {
+		case '\\':
+		    backslashed = true;
+		    break;
+
+		default:
+		    r += c;
+		    break;
+	    }
+	}
+	else
+	{
+	    switch (c)
+	    {
+		case 'n':
+		case 'l':
+		case 'r':
+		    r += '\n';
+		    break;
+
+		case '\\':
+		    r += '\\';
+		    break;
+	    }
```

Comment:
What about the `default` case?

https://graphviz.gitlab.io/_pages/doc/info/attrs.html#k:escString says:
>  In addition, if the associated attribute is label, headlabel or taillabel, the escape sequences "\n", "\l" and "\r" divide the label into lines, centered, left-justified, and right-justified, respectively.
> 
> Obviously, one can use "\\" to get a single backslash. *A backslash appearing before any character not listed above is ignored.*

So we should implement it with
```c++
		default:
		    r += c;
		    break;
```
That will also render the "label macros" like \G \N \E literally but that's fine, IMHO better than dropping them like now.

Reply:
AFAIS a default case would hurt here, the doc says "A backslash appearing before any character not listed above is ignored."

Reply:
Ah, I see. Your interpretation is "the backslash, and the following character, are ignored".
My interpretation was "only the backslash is ignored, the following character is rendered'.
It turns out graphviz follows my interpretation. Try
```sh
echo 'digraph g { "foo\t"; }' | dotty -
```